templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -241,52 +241,58 @@
+            {if $showCheckNumber || !$isOnline}
+              <tr id="checkNumber" class="crm-contribution-form-block-check_number">
+                <td class="label">{$form.check_number.label}</td>
+                <td>{$form.check_number.html}</td>
+              </tr>
+            {/if}
+            <tr class="crm-contribution-form-block-trxn_id">
+              <td class="label">{$form.trxn_id.label}</td>
+              <td {$valueStyle}>{$form.trxn_id.html} {help id="id-trans_id"}</td>
+            </tr>
+            {if $email and $outBound_option != 2}
+              <tr class="crm-contribution-form-block-is_email_receipt">
+                <td class="label">
+                  {$form.is_email_receipt.label}</td><td>{$form.is_email_receipt.html}&nbsp;
+                  <span class="description">{ts 1=$email}Automatically email a receipt for this payment to %1?{/ts}</span>
+                </td>
+              </tr>
+            {elseif $context eq 'standalone' and $outBound_option != 2 }
+              <tr id="email-receipt" style="display:none;" class="crm-contribution-form-block-is_email_receipt">
+                <td class="label">{$form.is_email_receipt.label}</td>
+                <td>{$form.is_email_receipt.html} <span class="description">{ts}Automatically email a receipt for this payment to {/ts}<span id="email-address"></span>?</span>
+                </td>
+              </tr>
+            {/if}
+            <tr id="receiptDate" class="crm-contribution-form-block-receipt_date">
+              <td class="label">{$form.receipt_date.label}</td>
+              <td>{include file="CRM/common/jcalendar.tpl" elementName=receipt_date}<br />
+                <span class="description">{ts}Date that a receipt was sent to the contributor.{/ts}</span>
               </td>
             </tr>
-          {/if}
-          <tr id="receiptDate" class="crm-contribution-form-block-receipt_date">
-            <td class="label">{$form.receipt_date.label}</td>
-            <td>{include file="CRM/common/jcalendar.tpl" elementName=receipt_date}<br />
-              <span class="description">{ts}Date that a receipt was sent to the contributor.{/ts}</span>
-            </td>
-          </tr>
-          <tr id="fromEmail" class="crm-contribution-form-block-receipt_date" style="display:none;">
-            <td class="label">{$form.from_email_address.label}</td>
-            <td>{$form.from_email_address.html}</td>
-          </tr>
-        </table>
-      </fieldset>
+            <tr id="fromEmail" class="crm-contribution-form-block-receipt_date" style="display:none;">
+              <td class="label">{$form.from_email_address.label}</td>
+              <td>{$form.from_email_address.html}</td>
+            </tr>
+          </table>
+        {/if}
+      </div>
+    </div>
   {/if}
 
   {include file='CRM/Core/BillingBlockWrapper.tpl'}
```

Comment:
Nope, it's needed for live mode. And won't affect the offline mode in addition to the new changes